### ISSUE TITLE
Misleading directions in "Sharing Fabricate Items"

### ISSUE THREAD

**Issue by cldebuys7** [SevenDayCandle/STS-CardEditor#1]
I'd like to start off by saying thanks for the mod; it's well written and incredibly useful. Now for my minor complaint.

In **Method 1: Traditional Mod (Recommended)** the directions suggest:

> To incorporate your custom items into a traditional mod, create a folder in your project's resources directory (usually located in src/main/resources subfolder) that contains a single type of entity (i.e. cards, relics, potions) along with their images. In your code, you will then want to call one of the following methods, depending on what type of entity you are loading:

This would lead the reader to believe `src/main/resources/cards` is the desired structure, for instance, or perhaps `src/main/resources/<modname>/cards` is the desired structure. The word choice suggests the former option, but the latter is more in keeping with typical mod structure. This second possibility is further enforced by the example given:

```
public BasicMod() {
        BaseMod.subscribe(this);
        logger.info(modID + " subscribed to BaseMod.");
        PCLCustomCardSlot.addProvider(modID, "basicmod/custom/cards"); // This line was added
    }
```

This code snippet, along with "along with their images" from the first direction, would lead the reader to believe that the card files and their images should be stored in `resources/basicmod/custom/cards`. However, _neither_ of these two options will work because `makeImagePath()`, which lives in `PCLCustomLoadable`, calls `getBaseFolderPath()`, and that method starts the file path with `"custom"`. So the only place the code will look for images is in `resources/custom`, regardless of what is given as input to `PCLCustomCardSlot.addProvider(modID, "path/to/card/files")`. The only time the user could store the card files and their images in the same folder is if that folder is `resources/custom`, which contradicts the provided example.

Long story short, there is ambiguity in the phrasing, and neither natural conclusion from the phrasing leads to the correct folder structure.

**Comment by SevenDayCandle** [comment]
Thanks for pointing that out. src/main/resources/<modname>/cards is indeed the intended folder for images and I will update that blurb to make that more apparent.

The image paths ignoring the path provided via addProvider is a bug; they're supposed to be sourced from the same folder as the files, and I'll work on a fix for that.

**Comment by SevenDayCandle** [comment]
The issue should be fixed with the latest patch I put out.
